config/initializers/elasticsearch.rb

```diff
@@ -13,22 +19,29 @@ def log(server, filepath)
 
 def create_es_cert_file(cert)
   begin
-    es_cert_file = Tempfile.new(%w(out .pem))
+    es_cert_file = File.new('elasticsearch_cert.pem', 'w')
```

Comment:
Why is the Tempfile class not being used anymore?

Reply:
Because the file gets deleted randomly. It's not too clear from the documentation when they're deleted, but we shouldn't assume they remain after we've closed them.